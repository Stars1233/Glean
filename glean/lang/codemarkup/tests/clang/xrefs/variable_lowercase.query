query: cxx1.VariableLowerCase _
transform: [gensort, []]
