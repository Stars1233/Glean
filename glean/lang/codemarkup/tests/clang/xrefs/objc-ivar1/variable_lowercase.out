[ "@generated", { "key": { "name_lowercase": "_i", "name": "_i" } } ]